portfolio.html

```diff
@@ -176,5 +176,85 @@ <h1>About me</h1>
         </ul>
       </div>
     </section>
+    <section id="contact">
+      <div>
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+      </div>
+      <div>
+        <form action="https://formspree.io/f/mnqwzykj" method="post">
+          <ul>
+            <li>
+              <label for="fullname"></label>
+              <input
+                type="text"
+                id="fullname"
+                name="user_name"
+                placeholder="Full name"
+                maxlength="30"
+                required
+              />
+            </li>
+            <li>
+              <label for="email"></label>
+              <input
+                type="email"
+                id="email"
+                name="user_email"
+                placeholder="Email address"
+                required
+              />
+            </li>
+            <li>
+              <label for="msg"></label>
+              <textarea
+                name="user_message"
+                id="msg"
+                cols="30"
+                rows="10"
+                placeholder="Enter text here"
+                maxlength="500"
+                required
+              ></textarea>
+            </li>
+            <li id="footer-button">
+              <button type="submit">Get in touch</button>
+            </li>
+          </ul>
+        </form>
```

Comment:
- [ ] Kindly ensure you stick to the design as much as possible. The form on the template is somewhat centered whereas your aligns more to the left except the button. Kindly fix this.

  Your Implementation | Figma Design
  :---:|:---:
  ![nil-or-1](https://user-images.githubusercontent.com/36542986/159536143-e4a27f0b-7dbd-4cf8-b362-fecaabe9075d.jpg) | ![nil-or-2](https://user-images.githubusercontent.com/36542986/159536153-8fc6a1f1-4676-4c03-949f-2b83133b6f6e.jpg)

Reply:
- [ ] [REQUIRED] Great effort so far, however, as recommended by the previous reviewer, you need to stick to the design template as much as possible. Kindly ensure that the `social-icons`, Get in touch `button`, the `input-fields` all get centered.
### Your Design Currently
![Screenshot from 2022-03-22 19-32-01](https://user-images.githubusercontent.com/58692590/159551679-25f556b7-bce2-4387-a183-a13736580794.png)